hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.js

```diff
@@ -270,6 +270,8 @@ define([
 
       function contractEdit() {
         $scope.$broadcast('hrjc-loader-show');
+        var period_end_date = $scope.entity.details.period_end_date;
```

Comment:
You missed updating the unit tests, please make sure to update unit tests for every JS change.